index.html

```diff
@@ -3,7 +3,48 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>Portfolio project 1</title>
+    <link
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css"
+      rel="stylesheet"
+    />
+    <link href="https://fonts.googleapis.com" rel="preconnect" />
+    <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:wght@100;300;400;500;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <link href="https://fonts.googleapis.com" rel="preconnect" />
+    <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin />
+    <link href="./css/index.css" rel="stylesheet" />
   </head>
-  <body></body>
+  <body class="body">
+    <div class="first-color">
+      <h1 class="owner">TENIFAYO ABDULSOMOD HABEEBULLAH</h1>
+      <span class="click">&#9776;</span>
```

Comment:
![Screenshot 2023-10-05 084724](https://github.com/abdulsomod1/portfolio-project/assets/105943931/ddffc51a-dae9-4777-9c23-b4f9baa07fa3)
yror harmburger icon seems too centered compared to ther figma design